mlflow/utils/openai_utils.py

```diff
@@ -140,6 +132,8 @@ class _OpenAIApiConfig(NamedTuple):
     api_base: str
     engine: Optional[str]
     deployment_id: Optional[str]
+    max_retries: int = 10
```

Comment:
Is there a particular reason you picked 10 for retry count here? It seems our original implementation used `5` and OpenAI [default](https://github.com/openai/openai-python/blob/52357cff50bee57ef442e94d78a0de38b4173fc2/src/openai/_constants.py#L10) is `2`.

Reply:
Sorry, this is for ci debug. Some tests in ci is flacky and I couldn't reproduce it. Definitely plan to set it to 5 to keep the existing setting.